resources/js/attributes.js

```diff
@@ -185,6 +185,11 @@ const listenToLiveForm = () => {
 
             let target = $('[data-uid="' + result.uid + '"]');
             //console.log('looking for', '[data-uid="' + result.uid + '"]', target);
+            if (result.checkbox == null) {
```

Comment:
We already do this if/else in PHP and now it's repeated in JS. Please refactor to only have the "logic" in one place, preferably PHP, by using a single variable ;-)